src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -637,6 +637,18 @@ public AppPage clickAndConfirm(WebElement elementToClick) {
     }
     
     /**
+     * Clicks the hidden element and clicks 'Yes' in the follow up dialog box.
+     * Fails if there is no dialog box.
+     * @return the resulting page.
+     */
+    public AppPage clickHiddenElementAndConfirm(String elementId) {
```

Comment:
I may be wrong, but I think this function can replace and decommission `clickAndConfirm` (similarly with `clickAndCancel`).